src/test/scala/org/ensime/intg/BasicWorkflow.scala

```diff
@@ -71,11 +71,8 @@ class BasicWorkflow extends FunSpec with Matchers {
         val javaSearchSymbol = project.rpcPublicSymbolSearch(List("java", "io", "File"), 2)
         javaSearchSymbol match {
           case SymbolSearchResults(List(
-            TypeSearchResult("java.awt.image.ImageObserver", "ImageObserver", 'class,
```

Comment:
if you do a search for `java io` I would expect `java.awt.image.ImageObserver` to come back (`io` matches camel case the `ImageObserver`). If you can add that as an additional search test I'm confident that there will be no regressions in expected behaviour.